app/Database/Repositories/Eloquent/UserRepository.php

```diff
@@ -136,4 +136,18 @@ public function findByUsername($username = '')
 	{
 		return $this->userModel->whereNname($username)->first();
 	}
+
+	/**
+	 * Create a new user
+	 *
+	 * @param array $details Details about the user.
+	 *
+	 * @return User
+	 */
+	public function create(array $details = [])
+	{
+		$user = $this->userModel->create($details);
```

Comment:
Can be simplified to `return $this->userModel->create($details);`, removing the intermediate variable.